### PR TITLE
Improve Accessibility: Portfolio Website

### PR DESCRIPTION
**Pull request by JEFFKAY16**
## Improve Accessibility: Portfolio Website

In this update, I was improving accessibility using the following guidelines: 

- Page titles - No issues found.
- Text headings - No issues found.
- Color contrast - Adjusted the [color](https://github.com/JEFFKAY16/Portfolio-Website/blob/bc124a155e1d23efb53b4e01bc75aed19b9559f5/styles.css#L99) of the social media icons to slightly darker theme so as to allow a good color contrast. 
- Resize - Social media icons were adjusted by increasing their [font-size](https://github.com/JEFFKAY16/Portfolio-Website/blob/bc124a155e1d23efb53b4e01bc75aed19b9559f5/styles.css#L100).
- Interaction - Improved the [project button's transition](https://github.com/JEFFKAY16/Portfolio-Website/blob/bc124a155e1d23efb53b4e01bc75aed19b9559f5/styles.css#L199) speed to be slightly slower so as to allow for a smooth transition.
- Moving content - No issues found.
- Multimedia - No issues found.
- The basic structure of the page -No issues found.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -96,7 +96,8 @@ footer p {
 }
```
Comment (SajjadAhmad14) [inline styles.css:96]: Please add a link to the corresponding line of code in the Pull Request If any issues are found and changes are made.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Blessed Mwanza and Jeff Kayombo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: styles.css (hunk 1 above)
Comment (blessedjasonmwanza) [reply styles.css:96]: Hello @SajjadAhmad14 ,

Thank you for the insightful review 🤓. We have added the links to the corresponding lines as per guide, in our pull request description.

> HappyCoding

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**